src/docker-compose.yml

```diff
@@ -16,7 +16,7 @@ services:
    image: "mojaloop/sdk-scheme-adapter:latest"
    env_file: ./scheme-adapter.env
    ports:
-     - "3500:3000"
-     - "3501:4000"
+     - "3500:4000"
```

Comment:
Please change this to `4000:4000`

Reply:
I have been using it with 4001/4000 without any issues. I suspect the issue is that there is a port conflict if you are running ml-api and central-ledger at the same time as they both use ports 4000 & 4001.

Reply:
I am ok to keep it 3500 & 3501 to avoid that conflicts.